### PR TITLE
MRG: Log all CoregistrationUI statusbar messages via logger.info() too

### PR DESCRIPTION
**Pull request by hoechenberger**
This makes it such that all messages we display on the status bar will also be sent to our logger (log level "info"). We had discussed adding some kind of logging system to the GUI in the past, and this here is just a very straightforward (cheap?) way to get some basic logging.

WDYT @GuillaumeFavelier @agramfort?

### CODE REVIEW

**Comment by agramfort** [comment]
yes 👍

**Comment by GuillaumeFavelier** [comment]
Indeed, `_display_message()` seems like a good place to me

**Comment by hoechenberger** [comment]
Cool, so let's get this in, and we can iterate in the future :)

File: mne/gui/_coreg.py [hunk 1]
```diff
@@ -855,6 +855,8 @@ def _display_message(self, msg=""):
         self._forward_widget_command(
             'status_message', 'update', None, input_value=False
         )
+        if msg:
```
Comment (hoechenberger) [inline mne/gui/_coreg.py:858]: Because the default is an empty string – we wouldn't want to log this

**Comment by larsoner** [comment]
Thanks @hoechenberger !

**Pull request merged**